lib/domain/service/smsshortify.ts

```diff
@@ -40,10 +39,16 @@ export class SMSShortify {
   }
 
   private createSuffix() {
-    this.suffix = this.link ? `${this.SEE_MORE} ${this.link}${this.suffix ? ` ${this.suffix}` : ''}` : undefined
+    if (!this.link) return
+    this.suffix = `${this.SEE_MORE} ${this.link}${this.suffix ? ` ${this.suffix}` : ''}`
```

Comment:
Não deveria ter um espaço entre o "link" e o "sufixo"?
```suggestion
    this.suffix = `${this.SEE_MORE} ${this.link} ${this.suffix ? ` ${this.suffix}` : ''}`
```

Reply:
No caso do teste era esperado o envio do sufixo com um "espaço" antes do "PEDIDO #123123-01", isso não é correto.

Reply:
feito, sem o ternário ajudou kkkk